### DISCUSSION TITLE
Build Flet Apk

### DISCUSSION THREAD

**Discussion by RafaelLacerda15** [flet-dev/flet#4299] [General]
PS C:\Users\rafae\Documents\Python_Projects\mobile\Animes Online> flet build apk --verbose
[16:26:51] Flutter bootstrap directory: C:\Users\rafae\AppData\Local\Temp\flet_flutter_build_zvlELb1tP3
           No additional Flutter dependencies!
[16:26:57] Created Flutter bootstrap project from gh:flet-dev/flet-build-template with ref 0.24.1 ✅
           Copying C:\Users\rafae\Documents\Python_Projects\mobile\Animes Online\assets\icon_android.png to C:\Users\rafae\AppData\Local\Temp\flet_flutter_build_zvlELb1tP3\images
           Copying C:\Users\rafae\Documents\Python_Projects\mobile\Animes Online\assets\splash.png to C:\Users\rafae\AppData\Local\Temp\flet_flutter_build_zvlELb1tP3\images
           Customized app icons and splash images ✅
           Run subprocess: ['C:\\flutter\\bin\\cache\\dart-sdk\\bin\\dart.EXE', 'run', 'flutter_launcher_icons']
(    ● ) Generating app icons (7.6s)
(   ●  ) Generating app icons (4.1s)
Got dependencies in `C:\Users\rafae\AppData\Local\Temp\flet_flutter_build_zvlELb1tP3`.
(●     ) Generating app icons (9.0s)
Built flutter_launcher_icons:flutter_launcher_icons.
(     ●) Generating app icons ⏳...   ════════════════════════════════════════════
     FLUTTER LAUNCHER ICONS (v0.13.1)
  ════════════════════════════════════════════

(    ● ) Generating app icons ⏳... • Creating default icons Android
• Overwriting the default Android launcher icon with a new icon
(●     ) Generating app icons ⏳... • Overwriting default iOS launcher icon with new icon
(  ●   ) Generating app icons ⏳... Creating Icons for Web...
(   ●  ) Generating app icons ⏳... Creating Icons for Windows...
(     ●) Generating app icons ⏳... Creating Icons for MacOS...
(  ●   ) Generating app icons ⏳... 
✓ Successfully generated launcher icons
[16:27:21] Generated app icons ✅
           Run subprocess: ['C:\\flutter\\bin\\cache\\dart-sdk\\bin\\dart.EXE', 'run', 'flutter_native_splash:create']
( ●    ) Generating splash screens (2.5s)
(●     ) Generating splash screens ⏳... Built flutter_native_splash:create.
(    ● ) Generating splash screens ⏳... [Android] Creating default splash images
[Android] Creating dark mode splash images
(   ●  ) Generating splash screens ⏳... [Android] Creating default android12splash images
[Android] Creating dark mode android12splash images
(  ●   ) Generating splash screens ⏳... [Android] Updating launch background(s) with splash image path...
[Android]  - android/app/src/main/res/drawable/launch_background.xml
(   ●  ) Generating splash screens ⏳... [Android]  - android/app/src/main/res/drawable-night/launch_background.xml
[Android]  - android/app/src/main/res/drawable-v21/launch_background.xml
[Android]  - android/app/src/main/res/drawable-night-v21/launch_background.xml
[Android] Updating styles...
[Android]  - android/app/src/main/res/values-v31/styles.xml
[Android] No android/app/src/main/res/values-v31/styles.xml found in your Android project
[Android] Creating android/app/src/main/res/values-v31/styles.xml and adding it to your Android project
[Android]  - android/app/src/main/res/values-night-v31/styles.xml
[Android] No android/app/src/main/res/values-night-v31/styles.xml found in your Android project
[Android] Creating android/app/src/main/res/values-night-v31/styles.xml and adding it to your Android project
[Android]  - android/app/src/main/res/values/styles.xml
[Android]  - android/app/src/main/res/values-night/styles.xml
[iOS] Creating  images
(    ● ) Generating splash screens ⏳... [iOS] Creating dark mode  images
[iOS] Updating ios/Runner/Info.plist for status bar hidden/visible
[Web] Creating images
(     ●) Generating splash screens ⏳... [Web] Creating images
(    ● ) Generating splash screens ⏳... [Web] Creating background images
[Web] Creating CSS
(   ●  ) Generating splash screens ⏳... [Web] Updating index.html
( ●    ) Generating splash screens ⏳... 
✅ Native splash complete.
Now go finish building something awesome! 💪 You rock! 🤘🤩
Like the package? Please give it a 👍 here: https://pub.dev/packages/flutter_native_splash

[16:27:26] Generated splash screens ✅
           Run subprocess: ['C:\\flutter\\bin\\cache\\dart-sdk\\bin\\dart.EXE', 'run', 'serious_python:main', 'package', 'C:\\Users\\rafae\\Documents\\Python_Projects\\mobile\\Animes      
           Online', '--mobile', '--platform', 'mobile', '--dep-mappings', 'flet=flet-embed', '--req-deps', 'flet-embed', '--exclude', 'build']
(   ●  ) Packaging Python app ⏳... 
Built serious_python:main.
( ●    ) Packaging Python app ⏳... Running package command
(  ●   ) Packaging Python app ⏳... Creating asset directory: C:\Users\rafae\AppData\Local\Temp\flet_flutter_build_zvlELb1tP3\app
Copying Python app from C:\Users\rafae\Documents\Python_Projects\mobile\Animes Online to C:\Users\rafae\AppData\Local\Temp\serious_python_temp6569472f
( ●    ) Packaging Python app ⏳... Configured mobile platform with sitecustomize.py at C:\Users\rafae\AppData\Local\Temp\serious_python_sitecustomizeaee5aff7\sitecustomize.py
Installing dependencies [flet-embed== 0.24.1, requests== 2.32.3] with pip command to C:\Users\rafae\AppData\Local\Temp\serious_python_temp6569472f\__pypackages__
(  ●   ) Packaging Python app ⏳... Extracting Python distributive from C:\Users\rafae\AppData\Local\Temp\cpython-3.11.6+20231002-x86_64-pc-windows-msvc-shared-install_only.tar.gz to C:\Users\rafae\AppData\Local\Temp\hostpython3.11_94729850
(  ●   ) Packaging Python app ⏳... Compiling Python sources at C:\Users\rafae\AppData\Local\Temp\serious_python_temp6569472f
[16:28:07] Deleting Flutter bootstrap directory C:\Users\rafae\AppData\Local\Temp\flet_flutter_build_zvlELb1tP3
           Error building Flet app - see the log of failed command above.
           Run subprocess: ['C:\\flutter\\bin\\flutter.BAT', 'doctor']
[16:28:18] Doctor summary (to see all details, run flutter doctor -v):
           [√] Flutter (Channel stable, 3.24.4, on Microsoft Windows , locale pt-BR)
           [√] Windows Version (Installed version of Windows is version 10 or higher)
           [√] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
           [√] Chrome - develop for the web
           [√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.11.5)
           [√] Android Studio (version 2024.1)
           [√] VS Code (version 1.83.1)
           [√] Connected device (3 available)
           [√] Network resources

           • No issues found!
           
This is happening when I use "Flet build apk", I've already changed the flet version to v0.25.0dev, I've already put Android in the latest version, I've already put Flutter in the latest version, but it didn't help, it's still the same . Could anyone help me?

**Comment by LegendaryPistachio** [comment]
The lack of official guidelines for matching Flet versions and Android app packaging dependencies may be the cause of your issue. Suggested solution: Use flet build apk -vv, which outputs logs and makes it easy to identify the reason for the packaging failure.

**Comment by LegendaryPistachio** [comment]
Your issue may be due to version incompatibility, as the Flet version and Android app packaging dependency versions are not matched. To troubleshoot, use flet build apk -vv, which will output logs and help you easily identify the cause of the packaging failure.

**Reply by RafaelLacerda15** [reply]
Thank you very much, I found the error, it was an api that had a backslash, it was unable to compile. Thank you very much once again.
